projects/core/src/occ/adapters/site-context/occ-site.adapter.ts

```diff
@@ -67,10 +68,14 @@ export class OccSiteAdapter implements SiteAdapter {
       );
   }
 
-  loadBaseSite(): Observable<BaseSite> {
+  /**
+   * There is no OCC API to load one site based on Uid.
+   * So, we have to load all sites, and find the one from the list.
+   */
+  loadBaseSite(siteUid?: string): Observable<BaseSite> {
     const baseUrl = this.occEndpointsService.getBaseEndpoint();
     const urlSplits = baseUrl.split('/');
-    const activeSite = urlSplits.pop();
+    const siteUidFromUrl = urlSplits.pop();
```

Comment:
Can we reuse the same configurable `'baseSites'` here? Actually, we might want to change this just leverage the `loadBaseSites()` method. Then we resolve the siteUid from URL if the siteUid argument is nullish.

Reply:
done